lib/vagrant-libvirt/config.rb

```diff
@@ -523,6 +525,17 @@ def smartcard(options = {})
         @smartcard_dev[:source_service] = options[:source_service] if @smartcard_dev[:type] == 'tcp'
       end
 
+      # Disk driver options for primary disk
+      def disk_driver(options = {})
+        @disk_driver_opts = {
+          cache: options[:cache],
+          io: options[:io],
+          copy_on_read: options[:copy_on_read],
+          discard: options[:discard],
+          detect_zeroes: options[:detect_zeroes]
+        }
+      end
```

Comment:
I think this may need some code added to the `merge` method to ensure that if someone was to define some of these settings in a `${HOME}/.vagrant.d/Vagrantfile` they would be merged with the disk_driver settings.

Though that would result in getting merged results which may cause some problems, but as you can set values to 'nil' explicitly, I think the benefits of allowing someone to provide settings in other Vagrantfiles to be used as defaults is worth it.

Reply:
Good call. I've added a commit to handle merging and added a note about setting the values to `nil` in the readme.